popout.js

```diff
@@ -821,7 +823,6 @@ class PopoutModule {
         if (game.keyboard.downKeys.has("Escape")) return; // eslint-disable-line no-undef
       }
       popout.close();
-      return oldClose.apply(app, args);
```

Comment:
Yes in theory this won't be called because the window will already be closed at this point, but logically it should still happen here. Was it causing problems for your module @typhonrt ?

Reply:
This causes the close method to be invoked twice and there is some odd side effect that causes TRL app windows to hang. You can remove this line as by calling `popout.close()` just above this will trigger the "unload" callback on `popout` and the old close method is invoked here: 
https://github.com/League-of-Foundry-Developers/fvtt-module-popout/blob/master/popout.js#L643

The double close callback happens when hitting the actual `close` button in the popped out app header.

Reply:
This issue remains.. Calling close twice in quick succession hangs TRL apps. As mentioned the changes in this PR don't affect core Foundry apps and the close callback is only invoked once.

Reply:
Can confirm this is still preventing the character sheets I made with TRL from working. Would be nice if we could get this fixed.